features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/listing/js/samples.js

```diff
@@ -81,10 +81,7 @@ APISamples.prototype.deploySample1 = function(){
 } 
 } 
 
-
-$(document).ready(function () {
-	$("#deploy_sample1").click(function(){
-		var deployer = new APISamples();
-		deployer.deploySample1();
-	});
-});
+var deploySampleApi = function(defaultTier){
```

Comment:
Please note that formatting issue has been corrected in current source [1].
[1] https://github.com/wso2/carbon-apimgt/blob/release-1.10.x/features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/listing/js/samples.js#L124